### PR TITLE
-Verbose should not override $ErrorActionPreference

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
Fix https://github.com/PowerShell/PowerShell/issues/2247

### CODE REVIEW

**Review by anmenaga — CHANGES_REQUESTED** [review]
This change is only for Verbose;
however, original issue says that it was approved for both: Verbose and Debug : "Committee agrees that we should fix this so that -ErrorAction works correctly and not overridden by -Debug or -Verbose".
Should this change be done for Debug as well?

**Comment by SteveL-MSFT** [comment]
@anmenaga that's a fair point about `-Debug`.  I'll update this.

**Comment by SteveL-MSFT** [comment]
@anmenaga can you review again?

File: test/powershell/Language/Scripting/ActionPreference.Tests.ps1 [hunk 1]
```diff
@@ -126,4 +126,23 @@
...
+            @{switch="-Debug"}
+        ) {
+            param($switch)
+            $ErrorActionPreference = "SilentlyContinue"
+            $params = @{
+                ItemType = "File";
+                Path = "$testdrive\test.txt";
+                Confirm = $false
+            }
+            New-Item @params
```
Comment (adityapatwardhan) [inline test/powershell/Language/Scripting/ActionPreference.Tests.ps1]: ` > $null`

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

File: test/powershell/Language/Scripting/ActionPreference.Tests.ps1 (hunk 1 above)
Comment (SteveL-MSFT) [reply test/powershell/Language/Scripting/ActionPreference.Tests.ps1]: will add

**Review by anmenaga — APPROVED** [review]

**Comment by SteveL-MSFT** [comment]
@adityapatwardhan can you review again?

**Review by adityapatwardhan — APPROVED** [review]

**Comment by mintsoft** [comment]
Is this likely to make it into the final for Powershell 6?

**Comment by markekraus** [comment]
@mintsoft This was merged to master so it will be in the release unless it needs to be reverted for some reason.

**Pull request merged**